tools/iceoryx_build_test.sh

```diff
@@ -185,8 +193,9 @@ while (( "$#" )); do
         echo "    build-all             Build all extensions and all examples"
         echo "    out-of-tree           Out-of-tree build for CI build"
         echo "    build-strict          Build is performed with '-Werror'"
-        echo "    build-shared         Build shared libs (iceoryx is build as static lib per default)"
+        echo "    build-shared          Build shared libs (iceoryx is build as static lib per default)"
         echo "    build-test            Builds all tests (doesn't run)"
+        echo "    test-add-user         Create additional useraccounts in system for testing access controll (default off)"
```

Comment:
Some typos: `Create additional user accounts in system for testing access control (default: off)"`